controllers/nodemaintenance_controller.go

```diff
@@ -55,6 +55,8 @@ const (
 	DrainerTimeout      = 30 * time.Second
 )
 
+var mgrConfig *rest.Config
```

Comment:
Please don't use package vars.

Reply:
I want to use the manager config for setting the helper struct client (as it was previously set), and the manager variable is only available in _SetupWithManager_ that used to set the drain helper struct once, and now it doesn't. To save this variable and set the helper struct client I use a package var (which is not ideal).

Do you have better alternatives? I can potentially pass this field in the NodeMaintenanceReconciler struct 🤔

Reply:
> Do you have better alternatives?

yes

> I can potentially pass this field in the NodeMaintenanceReconciler struct

this 🙂